src/Cdn/Cdn.Autorest/custom/Start-AzFrontDoorCdnProfilePrepareMigration.ps1

```diff
@@ -316,7 +316,7 @@ function Start-AzFrontDoorCdnProfilePrepareMigration {
 
             # 1. Enable MSI: get "principalId" from RP
             $commandArgs = @{ ResourceGroupName = ${ResourceGroupName}; Name = ${ProfileName}; IdentityType = ${IdentityType}; ErrorAction = 'Stop'}
-            if ($indentityType -ne "systemassigned") {
+            if ($IdentityType -ne "systemassigned") {
```

Comment:
Please do not change any code related names.

Reply:
@vidai-msft this was an edge case as i try not to touch code on these PRs, that will be later but this is 100% needs to be fixed if you look at line 318 it has the correct spelling so line 319 will never be actually evaluated.
Im actually worried that there are no pester tests to show the error that the value of the vaiable is not evaluated for "bad" values.